install_configure_sshd.sh

```diff
@@ -3,15 +3,20 @@
 if grep -q '#AuthorizedKeysCommand none' "$SSHD_CONFIG_FILE"; then
   sed -i "s:#AuthorizedKeysCommand none:AuthorizedKeysCommand ${AUTHORIZED_KEYS_COMMAND_FILE}:g" "$SSHD_CONFIG_FILE"
 else
-  if ! grep -q "AuthorizedKeysCommand ${AUTHORIZED_KEYS_COMMAND_FILE}" "$SSHD_CONFIG_FILE"; then
+  if ! grep -q "^AuthorizedKeysCommand ${AUTHORIZED_KEYS_COMMAND_FILE}" "$SSHD_CONFIG_FILE"; then
```

Comment:
If the value is set, but happens to be something else, then we probably shouldn't set it.